beta-src/src/components/WDActionIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { useState } from "react";
+import { SvgIcon, Box } from "@mui/material";
+import { ActionIcon, ActionIconSelected } from "./svgr-components";
+
+const WDActionIcon: React.FC = function () {
+  const [active, setActive] = useState(false);
+
+  const highlightActionHandler = () => {
+    setActive(!active);
+  };
+
+  return (
+    <Box
+      onClick={highlightActionHandler}
```

Comment:
i think we should avoid using onClick handlers on box because of accessibility reasons (like it won't be a semantic tag and i don't think you'll be able to tab to it). Isn't there an SVGIconButton available for this?